DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/ToSpeckle/WallConversionToSpeckle.cs

```diff
@@ -69,10 +69,11 @@ public override SOBR.RevitWall RawConvert(DB.Wall target)
 
     // POC : what to do if these parameters are unset (instead of assigning default)
     speckleWall.height =
-      _parameterValueExtractor.GetValueAsDouble(target, DB.BuiltInParameter.WALL_USER_HEIGHT_PARAM) ?? 0;
+      _parameterValueExtractor.GetValueAsDoubleOrNull(target, DB.BuiltInParameter.WALL_USER_HEIGHT_PARAM) ?? 0;
     speckleWall.baseOffset =
-      _parameterValueExtractor.GetValueAsDouble(target, DB.BuiltInParameter.WALL_BASE_OFFSET) ?? 0;
-    speckleWall.topOffset = _parameterValueExtractor.GetValueAsDouble(target, DB.BuiltInParameter.WALL_TOP_OFFSET) ?? 0;
+      _parameterValueExtractor.GetValueAsDoubleOrNull(target, DB.BuiltInParameter.WALL_BASE_OFFSET) ?? 0;
+    speckleWall.topOffset =
+      _parameterValueExtractor.GetValueAsDoubleOrNull(target, DB.BuiltInParameter.WALL_TOP_OFFSET) ?? 0;
```

Comment:
this seems odd, is a wall of height 0 even valid?

Wouldn't this be better as a `TryGet` pattern where you get both the result and the object? `bool TryGetValueAsDouble(target, ...WALL_TOP_OFFSET , out var theDouble)`